jenkins-jobs/scripts/config/FilteredNames.txt

```diff
@@ -0,0 +1,4 @@
+Jenkins user
+artiship
```

Comment:
Why are those three filtered out? I remember at least artiship being a real person?

Reply:
If either of you know what name to resolve these aliases to we can certainly make that connection but I was unable to find a connection by checking the commit history.

Reply:
So the one I found is that s0r01ed should be Sagar Rao. About the other two, I've just asked for their names in PRs they had created. Until we hear back I'd suggest we propagate the names as is to the list. Better than omitting them altogether IMO.

Reply:
Will do, I'll update the PR accordingly.